setup.py

```diff
@@ -1,3 +1,3 @@
 from setuptools import setup  # type: ignore[import]
 
-setup(name='asgiref')
+setup(name="asgiref")
```

Comment:
@andrewgodwin why is this here? it should work with just
```suggestion
setup()
```

Reply:
Presumably it was a copy-and-paste inheritance from an older setup.py. I think we can remove it.

Reply:
@andrewgodwin want me to land it in this PR?

Reply:
Sure